bitmovinanalyticsexample/src/main/java/com/bitmovin/bitmovinanalyticsexample/MainActivity.java

```diff
@@ -60,15 +63,16 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void initializeBitmovinPlayer(){
         PlayerConfiguration config = new PlayerConfiguration();
 
-        SourceConfiguration source = this.createSource();
+//        SourceConfiguration source = this.createSource();
+        SourceConfiguration source = this.createDRMSource();
         config.setSourceConfiguration(source);
 
-        AdvertisingConfiguration adConfig = initializeAds(config);
-        config.setAdvertisingConfiguration(adConfig);
+//        AdvertisingConfiguration adConfig = initializeAds(config);
+//        config.setAdvertisingConfiguration(adConfig);
```

Comment:
can you put commented block togehter so we now it belongs together ?

Reply:
https://github.com/bitmovin/bitmovin-analytics-collector-android/pull/80/commits/42efad3564b0eef0242516949bcedb2179414237